### DISCUSSION TITLE
How to wipe materialization for a single partition of an asset?

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#24534] [Q&A]
When I go to the asset details page and select a partition, if I then click `Wipe Materializations` then all partitions are wiped. Is it possible to do this currently?

**Comment by garethbrickman** [comment]
Not currently possible, but is on our radar to implement for Dagster OSS: https://github.com/dagster-io/dagster/issues/14749

**Comment by John-hyf** [comment]
Runs tab, filter with your specific partition, select fit records and click delete button within Actions dropdown
